README.md

```diff
@@ -121,6 +121,14 @@ locally if you are planning to contribute to the project.
   * For processing image files, `tesseract` is required. See [here](https://tesseract-ocr.github.io/tessdoc/Installation.html) for installation instructions.
   * For processing PDF files, `tesseract` and `poppler` are required. The [pdf2image docs](https://pdf2image.readthedocs.io/en/latest/installation.html) have instructions on installing `poppler` across various platforms.
 
+Additionally, if you're planning to contribute to `unstructured`, we provide you an optional `pre-commit` configuration
+file so that you make sure that your code matches the formatting and linting standards used in `unstructured`. Alternatively,
+you can use `make tidy` instead.
+
+If using the optional `pre-commit`, you'll need to install it as `pip install pre-commit`, and then install the
```

Comment:
I don't think `pip install`ing it separately is necessary if it's included in the `dev` dependencies. Maybe `pre-commit install`ing still is. Is that how we handle the optionality, install the python package but users have to install the hooks for it to activate?

Reply:
Another idea is to add a make target like `install pre-commit` that would both install the python package and run `pre-commit install`.

Reply:
Good catch, we don't need to tell the user explicitly `pip install` it. i think just having it installed with `make install` as part of a `dev.txt` requirement is fine. 

As for another `make` target for `pre-commit install` ... maybe OK to skip that one?

Somewhat related, the README could also mention that this can be removed with `pre-commit uninstall`.

Reply:
Thank you for your suggestions @qued and @cragwolfe I'll update that later today, so as to skip the explicit installation as can be done with `make install` and explain how to uninstall the hooks too 😄